main.sh

```diff
@@ -312,8 +312,6 @@ if [[ "$SYNC_GATEWAY" == 0 ]]; then
     --node-init-hostname="${LOCAL_IP}" \
     --node-init-data-path="${DISK_PATH}/data" \
     --node-init-index-path="${DISK_PATH}/index" \
-    --node-init-analytics-path="${DISK_PATH}/data" \
```

Comment:
Removing the values that are not supported in CC @malscent correct me if I am wrong here